java/arcs/core/util/performance/MemoryStats.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.util.performance
+
+/** Identify a memory consumption type with a description [desc]. */
+enum class MemoryIdentifier(val desc: String) {
+    TOTAL("Total memory usage in KB"),
+    JAVA_HEAP("The private Java Heap usage in KB"),
+    NATIVE_HEAP("The private Native Heap usage in KB"),
+    CODE_STATIC("The memory usage for static code and resources in KB"),
+    STACK("The stack memory usage in KB"),
+    GRAPHICS("The graphics memory usage in KB"),
+    SYSTEM("Shared and system memory usage in KB"),
+    OTHER("Other private memory usage in KB"),
+
+    /** Declare more per-platform/per-OS/per-purpose memory ids here. */
```

Comment:
Nit: this should probably just be a normal block comment, rather than a KDoc comment.

```kotlin
/* Declare more per-platform ..... */
```

(`/*` instead of `/**`)

Reply:
Oops, done.